apis/tempo/v1alpha1/tempostack_types.go

```diff
@@ -220,12 +220,18 @@ type ComponentStatus struct {
 
 // TempoStackStatus defines the observed state of TempoStack.
 type TempoStackStatus struct {
+	// Version of the Tempo Operator.
+	// +optional
+	OperatorVersion string `json:"operatorVersion,omitempty"`
```

Comment:
Why do we need this field? 

Can there be multiple operator versions per instance in the same cluster?

Reply:
This field is to identify which CRs need to be updated during the upgrade. It's required when we want to modify the CR without upgrading the tempo version.

The `tempoVersion` is only there for the end user, to have a nice output when running (`kubectl get tempo`).